src/interaction/ModifyTouch.js

```diff
@@ -14,17 +14,20 @@ import {boundingExtent as ol_extent_boundingExtent} from 'ol/extent'
  * @fires hidepopup
  * @extends {ol.interaction.Modify}
  * @param {olx.interaction.ModifyOptions} options
- *  @param {String|undefined} options.title title to display, default "remove point"
- *  @param {Boolean|undefined} options.usePopup use a popup, default true
+ * @param {String|undefined} options.title title to display, default "remove point"
+ * @param {String|undefined} options.className CSS class name for the popup
+ * @param {String|undefined} options.positioning positioning for the popup
+ * @param {Number|Array<number>|undefined} options.offsetBox offset box for the popup
+ * @param {Boolean|undefined} options.usePopup use a popup, default true
```

Comment:
It would be nice to also have an option for the [condition](https://github.com/openlayers/openlayers/blob/main/src/ol/interaction/Modify.js#L94) that is hard-coded few lines below.